README.md

```diff
@@ -1,65 +1,52 @@
-# codat-nextjs-sample-project
+# Codat Nextjs Sample Project
 
 ![Vercel](https://vercelbadge.vercel.app/api/mcclowes/codat-nextjs-sample-project)
-
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) and example requests from [Codat](https://docs.codat.io/docs).
-
-You can see it in action here: <https://codat-nextjs-sample-project.vercel.app/>
-
-This is a quick way of getting up and running with Codat's API, but any production application should have access control and other security measures to ensure your SMB customers' data is not exposed. 
-
-## What is Codat?
-
-Codat is the universal API for small business data. We provide real-time connectivity to enable software providers and financial institutions to build integrated products for their small business customers.
-
-Small businesses run on many different platforms. Codat gives these platforms a standard way to connect with each other so they can move data around seamlessly at scale.
+[![Deploy with Vercel](https://vercel.com/button)](deployWithVercel)
```

Comment:
Is this badge clear enough without context? (if they don't have a vercel account for instance?)

Reply:
Personally, I think yes. If someone doesn't know what vercel is I assume they'd just ignore it. I also think it's unlikely that your target audience would be wholey unfamilliar with vercel. I deffer to you here, though